### PR TITLE
Support e2e inputType

### PR DESCRIPTION
**Pull request by axinging**
Currently most benchamrks use image as input. When tuning performance,
it's hard to tell the bottleneck lies in the image (such as memory copy from CPU
to GPU, memory copy from GPU to GPU) or the GPU/CPU computing.

With inputType, we can easily understand the total performance (use
image) and the computing performance (use tensor) difference.

Example usage:
http://localhost:8080/local-benchmark/?backend=webgl&warmup=0&run=2&benchmark=bodypix&inputsize=0.25&architecture=MobileNetV1&inputType=image
inputType

inputType: image, tensor. This depends on model, check model_config.js.

FEATURE

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4651)

### CODE REVIEW

**Comment by axinging** [comment]
@qjia7 @haoyunfeix @xhcao, PTAL

File: e2e/benchmarks/model_config.js [hunk 1]
```diff
@@ -198,7 +198,10 @@ const benchmarks = {
     type: 'GraphModel',
     inputSizes: [128, 257, 512, 1024],
     architectures: ['MobileNetV1', 'ResNet50'],
-    load: async (inputResolution = 128, modelArchitecture = 'MobileNetV1') => {
+    inputTypes: ['image', 'tensor'],
```
Comment (qjia7) [inline e2e/benchmarks/model_config.js:201]: Can you also add imageBitmap as another option?

Comment (axinging) [reply e2e/benchmarks/model_config.js:201]: For below  models:
```
MobileNet, Bodypix, BlazeFace, PoseNet, coco-ssd, deeplab
```
They only support below input type, ImageBitmap is not includes:
```
tf.Tensor|ImageData|HTMLImageElement|HTMLCanvasElement|HTMLVideoElement
```
So before we add ImageBitmap support, we need to change the models input(from tfjs-models) first.

Comment (qjia7) [reply e2e/benchmarks/model_config.js:201]: Make sense for me. Thanks.

File: e2e/benchmarks/model_config.js [hunk 2]
```diff
@@ -216,7 +219,11 @@ const benchmarks = {
         };
       }
       const model = await posenet.load(config);
-      model.image = await loadImage('tennis_standing.jpg');
+      if (inputType === 'tensor') {
+        model.image = tf.zeros([inputResolution, inputResolution, 3]);
```
Comment (qjia7) [inline e2e/benchmarks/model_config.js:223]: nit: Prefer renaming to `model.input = ` rather than `model.image`. Same with other places.

**Review by qjia7 — COMMENTED** [review]
LGTM with a nit. Thanks.

**Comment by qjia7** [comment]
+@lina128 take a look, thanks.

File: e2e/benchmarks/model_config.js (hunk 2 above)
Comment (axinging) [reply e2e/benchmarks/model_config.js]: Done, PTAL.

**Review by lina128 — APPROVED** [review]
This is great! Thank you @axinging !

> *[Reviewable](https://reviewable.io/reviews/tensorflow/tfjs/4651#-:-MUA9tk0B1kuoKHo6tyr:b-ezopbm)* status: :shipit: complete! 1 of 1 approvals obtained (waiting on @axinging)

**Comment by axinging** [comment]
@lina128, please help to merge, thank!

**Pull request merged**